go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/scylladb/scylla-operator
 
-go 1.20
+go 1.21
```

Comment:
1.21.8

Reply:
Didn't want to mess with the order of backported commits, it will be bumped with the backport of https://github.com/scylladb/scylla-operator/pull/1806

Reply:
deps are not directly backportable for older branches, let's just do one

Reply:
done

Reply:
/hold I broke it, fixing now